project/FrontendBuild.scala

```diff
@@ -11,12 +11,16 @@ object FrontendBuild extends Build with MicroService {
 
   override lazy val appDependencies: Seq[ModuleID] = compile ++ test()
 
+  val parsebackVersion = "0.3"
+
   val compile = Seq(
     ws,
     "uk.gov.hmrc" %% "frontend-bootstrap" % "10.7.0",
     "uk.gov.hmrc" %% "auth-client" % "2.16.0-play-25",
     "uk.gov.hmrc" %% "domain" % "5.2.0",
     "uk.gov.hmrc" %% "play-partials" % "6.1.0",
+    "com.codecommit" %% "parseback-core" % parsebackVersion,
```

Comment:
I had to add 
```
.settings(resolvers ++= Seq(
   Resolver.jcenterRepo,
   "bintray-djspiewak-maven" at "https://dl.bintray.com/djspiewak/maven"
))
```
to `MicroService.scala` to make this branch compile.

Reply:
That's weird. It worked for me. I've added the resolvers anyway.

Reply:
I'm using coursier, so that may be the reason why it failed for me 🤔